BotNet.Services/SocialLink/SocialLinkEmbedFixer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace BotNet.Services.SocialLink {
+	public partial class SocialLinkEmbedFixer {
+		public static Uri Fix(Uri link) {
+			string url = link.ToString();
+			string newUrl = link.Host switch {
+				"twitter.com" => url.Replace("twitter.com", "vxtwitter.com"),
```

Comment:
```suggestion
				"twitter.com" => url.Replace("//twitter.com/", "//vxtwitter.com/"),
```
Lebih aman diapit slash